src/index.html

```diff
@@ -1,6 +1,19 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="preconnect"
+      href="D:\projects\layout_search-bar-airbnb\src\fonts"
+    />
+    <link
+      rel="preconnect"
+      href="D:\projects\layout_search-bar-airbnb\src\fonts"
```

Comment:
The `href` attribute for the `preconnect` link is pointing to a local file path. This should be a URL to a domain you want to preconnect to, not a local path.